css-three-column-layout/styles.css

```diff
@@ -0,0 +1,48 @@
+.container {
+  background-color: rgb(25 150 218);
+  color: white;
+  font-style: normal;
+  font-weight: 400;
+  margin: 0;
+}
+
+h1 {
+  display: flex;
+  justify-content: center;
+}
```

Comment:
You should be using `display: flex` on your `row` class, rather than applying it to this `h1`.